packages/webdriverio/src/commands/browser/setTimeout.ts

```diff
@@ -43,19 +43,20 @@ export default async function setTimeout(timeouts) {
 
     const implicit = timeouts.implicit
     // Previously also known as `page load` with JsonWireProtocol
-    const pageLoad = timeouts['page load'] || timeouts.pageLoad
+    const pageLoad = (timeouts as any)['page load'] || timeouts.pageLoad
```

Comment:
Shouldn't we extend the WebDriver.Timeout with our own types?

Reply:
`page load` is a JSONWireProtocol relict which I would not like to add to the typings given that we soon get rid of it anyway